src/rt/sched/cown.h

```diff
@@ -850,6 +850,9 @@ namespace verona::rt
     /// Return true if a sender to this cown should become low priority.
     inline bool triggers_muting()
     {
+      if (is_collected())
+        return false;
+
```

Comment:
If `is_collected()` is false, will that stay true at this point?

Reply:
It doesn't need to stay false in this execution since the next mute map scan would see that the mutor has been collected. I've moved the condition after the yield.